src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -224,44 +226,45 @@ class LedgerMasterImp
         return true;
     }
 
-    int getPublishedLedgerAge () override
+    std::chrono::seconds
+    getPublishedLedgerAge() override
     {
-        std::uint32_t pubClose = mPubLedgerClose.load();
-        if (!pubClose)
+        std::chrono::seconds pubClose{mPubLedgerClose.load()};
+        if (pubClose == 0s)
         {
             JLOG (m_journal.debug) << "No published ledger";
-            return 999999;
+            return weeks{2};
```

Comment:
I don't have a problem with this, but just highlighting the fact that we changed the return value max cap.